lib/migrator/methods/_loadConfig.js

```diff
@@ -35,11 +35,14 @@ module.exports = function _loadConfig(configPath) {
 					}
 				});
 			} else {
-				return '{}';
+				return {};
 			}
 		})
-		.then((configText) => {
-			const config = JSON.parse(configText);
+		.then((resultConfig) => {
+			const config = _.isString(resultConfig) ?
+				JSON.parse(resultConfig) :
+				resultConfig;
+
```

Comment:
I've added that condition, because config may not necessarily be in json format.

Reply:
There is a bug with config loading which introduced (apparently by me) in feature/v2 branch. I'll merge this PR and then will fix it properly.